javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
+
+/**
+ * adds two numbers togeter
+ * 
+ * @param {number} first the first number
+ * @returns {number}
+ */
+function add(first) {
+  /*
```

Comment:
Typo inside comment in token 'ere'. Possible corrections:pre (27),re (11),were (3)